### PR TITLE
switch base image - centos

### PR DESCRIPTION
**Pull request by joshfischer1108**

### CODE REVIEW

**Comment by joshfischer1108** [comment]
Base image repo can be found here: https://github.com/joshfischer1108/heron-centos7

**Comment by joshfischer1108** [comment]
With this change we should be able to release centos7 containers next release.

File: docker/dist/Dockerfile.dist.centos7 [hunk 1]
```diff
@@ -16,22 +16,10 @@
 #  under the License.
 
 #syntax=docker/dockerfile:1.2
-FROM centos:centos7
+FROM joshfischer1108/heron-centos7
```
Comment (nwangtw) [inline docker/dist/Dockerfile.dist.centos7:19]: This is expected? Official image should be better I feel?

Comment (joshfischer1108) [reply docker/dist/Dockerfile.dist.centos7:19]: I'm up for any suggestions.  We can create a company or organization under Dockerhub, and store the base image there.  Or something else if you prefer.

Comment (joshfischer1108) [reply docker/dist/Dockerfile.dist.centos7:19]: We can create an org like pulsar did https://hub.docker.com/r/apachepulsar/pulsar, but at the end of the day it's still just a dockerhub account.

**Pull request closed without merging**